README.md

```diff
@@ -1,18 +1,32 @@
-# The F# Language, Library, and  Visual F# Tools Repository
+# The F# language, Core library, and F# tools for Visual Studio
```

Comment:
I think it could be probably better to call it something like "F# editor tooling".
```suggestion
# The F# language, Core library, and F# editor tools
```